.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+charset = utf8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = tab
+indent_size = 4
+
+[{package.json,.travis.yml}]
+indent_style = space
+indent_size = 2
```

Comment:
Kinda funny that the file that says `insert_final_newline = true` as a rule, doesn't have a final newline.